master/pkg/model/experiment.go

```diff
@@ -477,6 +477,46 @@ type Trial struct {
 	Seed                  int64          `db:"seed"`
 	TotalBatches          int            `db:"total_batches"`
 	ExternalTrialID       *string        `db:"external_trial_id"`
+	RunID                 int            `db:"run_id"` // run_id as in restart_id not "runs" id.
+	LastActivity          *time.Time     `db:"last_activity"`
+}
+
+// ToRun converts a trial to a run.
+func (t *Trial) ToRun() *Run {
```

Comment:
write a test to make sure this is exhaustive? or turn the linter on for this struct?

Reply:
added a test